publish/publish_workflow.wdl

```diff
@@ -221,16 +233,14 @@ task prepGithub {
         echo 'Verify that the token has the scopes that will be required later'
 
         curl \
-            --fail --silent \
+            --location --fail --silent --show-error \
             --header "Authorization: token $(cat ~{githubTokenFile})" \
             --head \
             https://api.github.com/rate_limit \
-        | grep -F 'X-OAuth-Scopes:' > scopesHeader.txt
+        | grep -i -F 'X-OAuth-Scopes:' || true > scopesHeader.txt
```

Comment:
This combination of `--fail` and `|| true` seems strange to me?

Reply:
From the [ticket description](https://broadworkbench.atlassian.net/browse/BT-207):

>When searching for GitHub header names...update the grep arguments to ignore case and add a conditional statement to ignore failures. Error messages are generated in this block below and do not need to exit immediately when grep fails.